dotCMS/src/main/java/com/dotcms/contenttype/business/FieldApi.java

```diff
@@ -63,16 +159,51 @@ default FieldApi instance() {
 			Contentlet.HOST_KEY,
 			Contentlet.FOLDER_KEY);
 
-    Field byContentTypeIdAndVar(String id, String fieldVar) throws DotDataException;
-
-
-
+	/**
+	 * Returns a field based on the Content Type Id and the Field Variable
+	 * 
+	 * @param type Content Type Id to search on
+	 * @param fieldVar Field Variable to search on
+	 * @return Field Object that met the Field Variable and related to the Content Type
+	 * @throws DotDataException Error occurred when performing the action.
+	 */
+	Field byContentTypeIdAndVar(String id, String fieldVar) throws DotDataException;
+
+    /**
+	 * Deletes the field variable in the DB.
```

Comment:
"Deletes a field variable from a field (as specified in the passed in FieldVariable object)."